src/ripple/ledger/ReadView.h

```diff
@@ -45,10 +46,10 @@ namespace ripple {
 */
 struct Fees
 {
-    std::uint64_t base = 0;         // Reference tx cost (drops)
-    std::uint32_t units = 0;        // Reference fee units
-    std::uint32_t reserve = 0;      // Reserve base (drops)
-    std::uint32_t increment = 0;    // Reserve increment (drops)
+    XRPAmount base{ 0 };         // Reference tx cost (drops)
+    FeeUnit32 units{ 0 };        // Reference fee units
```

Comment:
I know you didn't change this, but does a reference fee of zero make semantic sense?

Reply:
I suspect the intention of initializing to 0 is just to prevent it from being uninitialized, and so that if it is used, there will be a very attention getting divide-by-0, instead of meaningless values. The values are set in `Ledger::setup` before it's ever used.

Reply:
I suspect the intention of initializing to 0 is just to prevent it from being uninitialized, and so that if it is used, there will be a very attention getting divide-by-0, instead of meaningless values. The values are set in `Ledger::setup` before it's ever used.

Reply:
If something doesn't have a reasonable default state, I usually delete the default constructor. I can see how the two phase setup in Ledger complicates this. No change needed (it's beyond the scope of this PR to fix this, and I don't have an easy fix anyway). Marking this as resolved.